types/kue/index.d.ts

```diff
@@ -63,7 +64,9 @@ interface Priorities {
 
 export type DoneCallback = (err?: any, result?: any) => void;
 export type JobCallback = (err?: any, job?: Job) => void;
-export type ProcessCallback = (job: Job, cb: DoneCallback) => void;
+export type ProcessCallback =
+    | ((job: Job, cb: DoneCallback) => void)
+    | ((job: Job, ctx: WorkerCtx, cb: DoneCallback) => void);
```

Comment:
should we need to add a test regarding the changes?

Reply:
Yes, that's a good idea. I added the pausing example from the readme as a test